routes/project.js

```diff
@@ -148,7 +149,7 @@ function getProjectStats(req, res, next) {
  * }
  */
 function createProject(req, res, next) {
-  const validBodyAttrs = ['name', 'metadata'];
+  const validBodyAttrs = ['name', 'quadkeySetId', 'metadata'];
```

Comment:
@batpad I think you want to use `quadkey_set_id` instead of `quadkeySetId ` here.